### PR TITLE
[JENKINS-29542] Escaping '$' to be not parsed in jelly help page

### PR DESCRIPTION
**Pull request by rodrigozrusso**

### CODE REVIEW

File: cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/EnvActionImpl/Binder/help.jelly [hunk 1]
```diff
@@ -41,7 +41,7 @@ node {
     </p>
     <p>
         <pre>mail to: 'devops@acme.com',
-    subject: "Job '${env.JOB_NAME}' (${env.BUILD_NUMBER}) is waiting for input",
+    subject: "Job '$${env.JOB_NAME}' ($${env.BUILD_NUMBER}) is waiting for input",
     body: "Please go to ${env.BUILD_URL} and verify the build"</pre>
```
Comment (jglick) [inline cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/EnvActionImpl/Binder/help.jelly]: What about this line?

**Comment by mrroohian** [comment]
Without this change ${env.JOB_NAME} and ${env.BUILD_NUMBER} results in empty string in the snippet generator help section:

mail to: 'devops@acme.com',
    subject: "Job '' () is waiting for input",
    body: "Please go to  and verify the build"

**Comment by mroohian** [comment]
Without this change ${env.JOB_NAME} and ${env.BUILD_NUMBER} results in empty string in the snippet generator help section:

mail to: 'devops@acme.com',
subject: "Job '' () is waiting for input",
body: "Please go to and verify the build"

**Comment by jglick** [comment]
> body: "Please go to and verify the build"

So line 45 indeed needs to be patched too.

File: cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/EnvActionImpl/Binder/help.jelly (hunk 1 above)
Comment (KostyaSha) [inline cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/EnvActionImpl/Binder/help.jelly]: seems jelly as xml escapes this special char.

**Pull request merged**